Week3/hw/app6.js

```diff
@@ -0,0 +1,9 @@
+//  6. Create a function called `vehicle`, like before, but takes another parameter called `age`, so that `vehicle("blue", 1, 5)` prints _'a blue used car'_
+
+function vehicle(color, code, age) {
+  if (('blue', 1, age > 5)) {
```

Comment:
Same here, I don't think this is correct and I don't think that this works.. can you look at this again?